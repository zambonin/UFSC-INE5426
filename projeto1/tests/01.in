  int a_
int	BB, c
int d=   0,	e1 =1

a_ = d+2* 3
BB = (-a_)	/  12-1
	c = e1*e1/a_


int Abra = 1, Kadabra, Alakazam

Kadabra = Abra * 10 + 1
Alakazam = 1 + Kadabra * 10

int x10_12Y
x10_12Y = - (10 + 2) * (3 - 4) / (-0 + 1)

