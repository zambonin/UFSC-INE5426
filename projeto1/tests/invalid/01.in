
int a
int	####b $#@#%%
int a, a
int 120b
int
ddsdsa


A = 2
int Abra = 1, Kadabra, Alakazam

abra = Kadabra * 10
