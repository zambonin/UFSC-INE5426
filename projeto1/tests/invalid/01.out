[Line 3] lexical error: unknown symbol ####
[Line 3] lexical error: unknown symbol $#@#%%
[Line 4] semantic error: re-declaration of variable a
[Line 5] semantic error: re-declaration of variable a
[Line 5] syntax error[Line 7] syntax error[Line 8] syntax error[Line 11] semantic error: undeclared variable A
[Line 14] semantic error: undeclared variable abra
